### ISSUE TITLE
In 2022, can't get examples to work in

### ISSUE THREAD

**Issue by seanw2020** [bmuschko/gradle-in-action-source#13]
First, thank you for writing this book!

Is it possible to run the code examples in a recent version of IntelliJ IDEA? I'm using 2021.3 Ultimate Edition. I'm using the gradle wrapper with the version set (see image below, note that the `s` is now required unlike a few years ago) as you suggest in your README, and I followed the tips you mentioned in [this issue](https://github.com/bmuschko/gradle-in-action-source/issues/7), but I get this output:

```
11:32 PM	Gradle sync failed: This version of Android Studio requires projects to use Gradle 4.8.1 or newer. This project is using Gradle 1.7.
			Change to minimum versions (plugin 3.2.0, Gradle 4.8.1) and sync project
			Change to latest versions (plugin 7.0.0, Gradle 7.0.2) and sync project
			Open build file (16 s 601 ms)
```

My settings look like this:

![image](https://user-images.githubusercontent.com/18521457/153129019-92a8a93f-a115-458e-9138-13a9633c7fe4.png)
![image](https://user-images.githubusercontent.com/18521457/153128564-e8d266a8-f907-48b4-9850-5de4bf4fdea8.png)
![image](https://user-images.githubusercontent.com/18521457/153128617-847e4f27-53d2-4ba3-a71b-38b75969f1ad.png)
![image](https://user-images.githubusercontent.com/18521457/153128664-a78d68cb-58f2-40d6-b4e8-bc21cfc881c5.png)

Now that it's 2022, should I just give up on getting the code to work, and instead follow the book content as best as possible, following up with something like the Udemy [Gradle Masterclass](https://www.udemy.com/course/gradle-masterclass/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-HIWxRjHEMWYPe6B.AUMC6Q&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads)?

**Comment by bmuschko** [comment]
Thanks! You are using Android Studio and not IntelliJ IDEA which seems to enforce a specific Gradle version. Try this with [IntelliJ IDEA community edition](https://www.jetbrains.com/idea/download).

**Comment by seanw2020** [comment]
Cool, that gets me closer, but rather than installing Community Edition, it turns out that you can disable IntelliJ's Android Studio plugin to achieve the same thing...

![image](https://user-images.githubusercontent.com/18521457/153229813-95224cc5-25ea-490a-a47e-f25a3ce75914.png)

..., but then I get this:

>Unsupported Gradle. 
>The project uses Gradle 1.7 which is incompatible with IntelliJ IDEA 2021.3.
>Possible solution:
>- Upgrade Gradle wrapper to 3.0 version and re-import the project

So it seems IntelliJ integration is impossible.

As a workaround, should I just run the gradle commands from IntelliJ's terminal? For example, using sdk-man, I can select gradle 1.7, and cd to the project's directory.

**Comment by bmuschko** [comment]
You can use a newer version than Gradle 1.7 and I am sure the IDE would still be happy with it. In addition, Gradle now also requires you to add a `settings.gradle` file for each project. The way you opened the "project" in IntelliJ isn't correct as you are not dealing with a large multi-project build. You need to open projects individually e.g. `chapter02/helloworld-task-doLast`.

Side note: For learning purposes, you don't really need an IDE. You may just open a project from an editor e.g. VsCode. You can always execute the build from the command line.

Bottom line: The book would need a major overhaul, however, Manning Publications decided not to work on a second edition.
